utils/notifications.js

```diff
@@ -110,18 +110,26 @@ const markAllNotificationsAsAlreadyRead = (notification, collection) => {
 }
 
 const retrieveNotifications = async (req, res, uid) => {
+  if (req.method !== "GET") {
+    return res.status(405).json(getResponseJSON("Only GET requests are accepted!", 405));
```

Comment:
unrelated - but thoughts on creating a shared function that can be utilized for common non-200 statuses such as 405 and 500? We could ensure consist messaging and formatting of these responses.

Reply:
Agree. We should discuss and do it.

Reply:
was a new index required? if so, we will want to remember when code goes to each environment